### ISSUE TITLE
explaining TE output

### ISSUE THREAD

**Issue by mars188** [wyang17/SQuIRE#66]
Hello, 

I have outputted a file called "DESeq_TE_only.txt" and it looks fine but I am confused about the gene/TE names in the first column. For example, they look like this:

`chr22|36735837|36736195|HE2_DR:tRNA-V:SINE|42|+,.`
`chr1|25168232|25169679|TZF28B:TcMar-Tc1:DNA|78|-,.`
`chr7|56584065|56584886|ERV1-N4-LTR_DR:LTR:LTR|17|+,.`
`chr21|22685323|22686081|L2-34_DR:L2:LINE|128|-,.`

I think actual TE name is "HE2_DR:tRNA-V:SINE". Is there any script to separate LINE, SINE, DNA and LTR classes of TEs?

**Comment by mars188** [comment]
Also, would be great what each column represents? like for the first row:
chr22: ?
36735837: ?
36736195: ?
HE2_DR:tRNA-V:SINE: ??

And what are 42, 78, 14 and 128 at the end. Also, what does +/- sign at the end indicate? 

Many thanks!

**Comment by mars188** [comment]
Also, I get TE and gens both mixed with each other in a single file. Is that normal?

There are two files of interest but I am confused what's the difference between them:

DESeq2_all: It contains genes and TE together
DESeq2_TE_only: I thought it should contain only TEs but actually I see genes and TE together in this file. 

Can you please help me understand the output results?

Many thanks,

**Comment by Surar** [comment]
Hi @mars188 have you found a way to separate LINE, SINE, DNA and LTR classes of TEs?
Thanks.

**Comment by mars188** [comment]
@Surar yes I did. I restarted with a freshly installed squire in a new/clean environment, and that solved the issues.
